src/silo/preprocessing/preprocessor.test.cpp

```diff
@@ -53,6 +53,35 @@ const Scenario FASTA_FILES_WITH_MISSING_SEGMENTS_AND_GENES = {
 }])")
 };
 
+// First 10 characters of sequence in nuc_main.sam have been removed
+// This is to test the offset (set to 10 for both reads)
+const Scenario SAM_FILES = {
+   .input_directory = "testBaseData/samFiles/",
+   .expected_sequence_count = 2,
+   .query = R"(
+      {
+         "action": {
+           "type": "FastaAligned",
+           "sequenceName": ["main"],
+           "orderByFields": ["accessionVersion"]
+         },
+         "filterExpression": {
+            "type": "True"
+         }
+      }
+   )",
+   .expected_query_result = nlohmann::json::parse(R"([
+         {
+            "accessionVersion": "1.1",
+            "main": "NNNNNNNNNNTATACCTTCCCAGGTAACAAACCAACCAACTTTCGATCT"
+         },
+         {
+            "accessionVersion": "1.3",
+            "main": "NNNNNNNNNNTATACCTTCCCAGGTAACAAACCAACCAACTTTCGATCT"
+         }
+      ])")
+};
```

Comment:
Let's maybe have one sequence with less offset, and maybe a bit shorter, so that we can see that the end of the sequence is also filled with Ns?